### ISSUE TITLE
cli demo code？

### ISSUE THREAD

**Issue by JoshonSmith** [LituRout/RF-Inversion#3]

**Comment by zethfoxster** [comment]
yeah i would like to see cli code if you could, this is fantastic

**Comment by JoshonSmith** [comment]
https://github.com/raven38/rf_inversion

**Comment by DarkMnDragon** [comment]
https://github.com/DarkMnDragon/rf-inversion-diffuser

Yeah, I also reimplemented a notebook, you can try that

**Comment by pureexe** [comment]
I'm currently working on SD3.  If you aim to use RF-inversion SD3 or SD3.5, you can take my implementation. 

https://github.com/pureexe/rf-inversion-sd3

Also, thank for DarkMnDragon give an example of how to implement on Flux
